### DISCUSSION TITLE
Copy and paste from Excel

### DISCUSSION THREAD

**Discussion by akheel147** [KevinVandy/material-react-table#160] [Q&A]
Is there any way to copy data from Excel sheet and paste it into the material react table

**Comment by KevinVandy** [comment]
No, not built in, but you could do something manually with an onPaste that mutates your data

**Reply by tobiasBora** [reply]
Is there any plan to support this use case? It would also be really handy to copy/paste stuff inside react table itself (either inside the same table, our from table 1 to table 2). For now the only alternative I found is revogrid but it has poor react support.

**Reply by KevinVandy** [reply]
MRT is not necessary meant to be a full excel replacement, so not sure if this will get added soon. AG Grid might be more what you are looking for.

**Reply by tobiasBora** [reply]
Thanks, too bad. I’m trying to use only open source softwares, and AG-grid’s cell selection is in the commercial product only. My goal is not to replace excel, but to easily copy/paste cells from one table to another, or to import/export the result to excel. I guess I’ll code my own solution based on Ant Design.

**Reply by KevinVandy** [reply]
It's probably possible to accomplish some of that stuff. Click to copy and editing are already built in features. All cell props are exposed for custom events. MRT is just not built to act like Excel.
